article-analyzer/src/main/java/co/inventorsoft/academy/spring/service/ArticleService.java

```diff
@@ -0,0 +1,36 @@
+package co.inventorsoft.academy.spring.service;
+
+import co.inventorsoft.academy.spring.repository.ArticleRepository;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.stream.Stream;
+
+@Data
+@RequiredArgsConstructor
+@Service
+public class ArticleService {
+
+    @Value("${helper.words}")
+    private List<String> helperWords;
+
+    private final ArticleRepository articleRepository;
+
+    public List<List<String>> splitArticleWords() throws FileNotFoundException {
```

Comment:
![NiceGIF](https://github.com/vitaliiStefanchakInventorsoft/spring-lecture/assets/103140969/2bed6cce-253b-4012-a4e7-3d6c69d38b72)

The logic is cool, but catch exception here

Also rename the method name to getSplittedArticleWords or something like that, because from this name It suggests that method should be void

Reply:
changed